mailbox/api/src/main/java/org/apache/james/mailbox/model/UpdatedFlags.java

```diff
@@ -104,7 +143,19 @@ private static boolean isChanged(Flags original, Flags updated, String userFlag)
     public Flags getOldFlags() {
         return oldFlags;
     }
-    
+
+    public boolean isSet(Flags.Flag flag) {
```

Comment:
method name looks misleading, UpdatedFlags.isSet(recent) doesn't tell me that the recent flag changed from false to true.